improver_tests/acceptance/SHA256SUMS

```diff
@@ -1,90 +1,96 @@
-047420170787241db82c89529c63fd6631533c163830a3fa3d9b8363af6958aa  ./aggregate-reliability-tables/basic/collapse_lat_lon_kgo.nc
-a6a84d0142796e4b9ca7bd3f0ad78586ea77684f5df02732fdda2ab54233cbb6  ./aggregate-reliability-tables/basic/multiple_tables_kgo.nc
+7b09ea9c0d798eb9d95496262f1b2328ee64b0ef0664d063fc540450b88bc880  ./aggregate-reliability-tables/basic/collapse_lat_lon_kgo.nc
```

Comment:
I guess this all needs merging / rebasing on master following the environment upgrade work?